### PR TITLE
draft: added community documentation

### PR DESCRIPTION
**Pull request by tibeer**
Documentation regarding tools used to interact with the community and how to get onboarded added.

Signed-off-by: Tim Beermann <beermann@osism.tech>

### CODE REVIEW

**Comment by tibeer** [comment]
@itrich please look into this.

File: Community/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- provide ONE single URL for all services with proper subdomains:
+  - nextcloud.sovereignit.de
+  - lists.sovereignit.de
+  - jitsi.sovereignit.de
+  - sovereignit.de (website)
+  - I've never understood why there is scs.community and scs.sovereignit.de
+- I would love to have a single tool for chat and video calls (slack, zulip, rocket chat, mattermost, element matrix, microsoft teams, i don't care)
+- overall documentation is too much distributed. I would expect this "https://scs.community/docs/contributor/" to be available as a menu point directly in "https://scs.community/". AND: I would expect it to be in the same design style. Would make things feel much more smooth.
+- discussion point: add an "onboarding" channel in the desired instant messaging tool
+- tell me how i can help to organize stuff, happy to contribute!
```
Comment (itrich) [inline Community/README.md:47]: I suggest to establish a Community Board and discuss all these proposals on a monthly basis.

File: Community/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- there has been done a lot of work by you guys which is really great. And even thouh I critisied a lot, I would still say thank you for your hard work. It's hard to write good documentation, your's is very good!
```
Comment (itrich) [inline Community/README.md:48]: Thank you! :heart:

File: Community/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- don't spam the mailing lists with complete annoucements, but rather the link to the annoucement!
+- ditch Wekan and move to GitHub to reduce tool clutter
+- provide ONE single URL for all services with proper subdomains:
+  - nextcloud.sovereignit.de
+  - lists.sovereignit.de
+  - jitsi.sovereignit.de
+  - sovereignit.de (website)
+  - I've never understood why there is scs.community and scs.sovereignit.de
+- I would love to have a single tool for chat and video calls (slack, zulip, rocket chat, mattermost, element matrix, microsoft teams, i don't care)
+- overall documentation is too much distributed. I would expect this "https://scs.community/docs/contributor/" to be available as a menu point directly in "https://scs.community/". AND: I would expect it to be in the same design style. Would make things feel much more smooth.
```
Comment (itrich) [inline Community/README.md:45]: This is something we want to tackle within the next months. IMHO all documentation should be reside in the `SovereignCloudStack/Docs` repository. This repository should then be used to generate a website by using a static site generator. I'll have a look if we can easily match the `website` Jekyll framework with the `Docs` repository.

File: Community/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- discussion point: add an "onboarding" channel in the desired instant messaging tool
```
Comment (itrich) [inline Community/README.md:46]: That's a great idea! :+1:

File: Community/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Community
+
+Entrypoint for getting onboarded.
+
+## Tools used by the community
+
+- Instant messaging
+  - [Matrix](https://matrix.to/#/!TiDqlLmEUaXqTemaLc:matrix.org?via=matrix.org)
+  - [Nextcloud Talk](https://scs.sovereignit.de/nextcloud) (Not used)
```
Comment (itrich) [inline Community/README.md:9]: If Talk is not used, we shouldn't list it here either, or should we?

File: Community/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  - [Fosstodon](https://fosstodon.org/@sovereigncloudstack) (Only one-directional)
+- Annoucements
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+  - [RSS Feed](https://scs.community/feed.xml)
+  - [Footodon](https://fosstodon.org/@sovereigncloudstack)
+  - [Twitter](https://twitter.com/scs_osballiance)
+  - [LinedIn](https://www.linkedin.com/showcase/sovereigncloudstack)
```
Comment (fkr) [inline Community/README.md:16]: [LinkedIn]

(typo)

File: Community/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+## Tools used by the community
+
+- Instant messaging
+  - [Matrix](https://matrix.to/#/!TiDqlLmEUaXqTemaLc:matrix.org?via=matrix.org)
+  - [Nextcloud Talk](https://scs.sovereignit.de/nextcloud) (Not used)
+  - [Fosstodon](https://fosstodon.org/@sovereigncloudstack) (Only one-directional)
+- Annoucements
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+  - [RSS Feed](https://scs.community/feed.xml)
+  - [Footodon](https://fosstodon.org/@sovereigncloudstack)
```
Comment (fkr) [inline Community/README.md:14]: [Fosstodon]

(typo)

File: Community/README.md (hunk 5 above)
Comment (tibeer) [reply Community/README.md:9]: Depends, technically, we have the tool available. Not sure.

Comment (frosty-geek) [reply Community/README.md:9]: IMHO, we should focus on matrix as gaia-x uses it as well.

File: Community/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  - [Contributor Guide](https://scs.community/docs/contributor/)
+
+### Proposal to improve (Just my two cents)
+
+- reduce usage of mailing lists drastically
+- reduce amount of mailing lists
+- use single source of annoucement
+- don't spam the mailing lists with complete annoucements, but rather the link to the annoucement!
+- ditch Wekan and move to GitHub to reduce tool clutter
+- provide ONE single URL for all services with proper subdomains:
```
Comment (frosty-geek) [inline Community/README.md:38]: my 2 cents, having a single well-known entry point /portal is fine for me.

File: Community/README.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  - nextcloud.sovereignit.de
+  - lists.sovereignit.de
+  - jitsi.sovereignit.de
+  - sovereignit.de (website)
+  - I've never understood why there is scs.community and scs.sovereignit.de
```
Comment (frosty-geek) [inline Community/README.md:43]: sovereignit.cloud is also in use for the gx-scs demonstrator.

maybe something like "debian.org vs. debian.net"... sorry I can't find the page where it pointed you to when to use what domain.

https://wiki.debian.org/MemberBenefits

**Comment by fkr** [comment]
My suggestion is:

Add this as REDME.md to the contributor-guide directory. That is still missing a readme and this is an awesome entry point to the contributor guide.
Since I need to document the discussion regarding backporting (#60) I'd like to build on top of that.

**Review by fkr — APPROVED** [review]
Further items can be improved via further PRs.

**Pull request merged**